rpcserver.go

```diff
@@ -6098,8 +6098,15 @@ func (r *rpcServer) GetTransactions(ctx context.Context,
 		endHeight = req.EndHeight
 	}
 
+	// To remain backward compatible, If the number of transactions was not
+	// specified, then we'll default to returning the entire transactions
+	if req.MaxTransactions == 0 {
```

Comment:
I think this can be replaced by the use of a default value. See comment above.

Reply:
I added the default value of -1 when max_transactions is not specified from the CLI. However, for RPC, the default appears to be 0, so I am checking and replacing it with -1. I used [this](https://github.com/lightningnetwork/lnd/blob/acbb33bb7b11e2dd3f2f85fe14fd3ff087212fc0/rpcserver.go#L3591) as a reference.

Reply:
Right, RPC users will have a default value of 0, makes sense then.